src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/BackgroundDocumentGenerator.cs

```diff
@@ -155,16 +156,13 @@ internal void Enqueue(DocumentSnapshot document)
         private void StartWorker()
         {
             // Access to the timer is protected by the lock in Synchronize and in Timer_Tick
-            if (_timer is null)
-            {
-                // Timer will fire after a fixed delay, but only once.
-                _timer = new Timer(Timer_Tick, null, Delay, Timeout.InfiniteTimeSpan);
-            }
+            // Timer will fire after a fixed delay, but only once.
+            _timer ??= new Timer(Timer_Tick, null, Delay, Timeout.InfiniteTimeSpan);
         }
 
-        private void Timer_Tick(object state)
+        private void Timer_Tick(object? state)
         {
-            _ = Timer_TickAsync(CancellationToken.None);
+            Timer_TickAsync(CancellationToken.None).Forget();
```

Comment:
📝 `Forget()` is a nice helper provided by the VS threading library: https://github.com/microsoft/vs-threading/blob/c188154eb90600e0b0b78c194e94471137111218/src/Microsoft.VisualStudio.Threading/TplExtensions.cs#L244-L265.